### PR TITLE
 Changing the task version to V0 and V1 respectively

### PR DESCRIPTION
**Pull request by fadnavistanmay**

### CODE REVIEW

**Review by owenhuynMSFT — APPROVED** [review]

**Review by owenhuynMSFT — APPROVED** [review]

File: Tasks/DownloadPipelineArtifactV1/task.loc.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 1
+    "Patch": 0
```
Comment (stephenmichaelf) [inline Tasks/DownloadPipelineArtifactV1/task.loc.json:12]: Why bump the patch down?

File: Tasks/DownloadPipelineArtifactV1/task.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 1
+        "Patch": 0
```
Comment (stephenmichaelf) [inline Tasks/DownloadPipelineArtifactV1/task.json:12]: Why bump the patch down?

File: Tasks/DownloadPipelineArtifactV1/task.loc.json (hunk 1 above)
Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV1/task.loc.json:12]: Hi @stephenmichaelf , The original version should be 0. We had increased the patch , just for testing purpose.

File: Tasks/DownloadPipelineArtifactV1/task.json (hunk 2 above)
Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV1/task.json:12]: @stephenmichaelf , same as above.

**Pull request merged**